### PR TITLE
TELCODOCS-1511-RN adding all-multicast mode RN

### PR DESCRIPTION
**Pull request by kquinn1204**
[TELCODOCS-1511]: [CNF-8804](https://issues.redhat.com//browse/CNF-8804) Tuning CNI- ALLMULTI flag (GA) 

Version(s):4.15 

Issue:https://issues.redhat.com/browse/TELCODOCS-1511

Link to docs preview: https://70724--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-all-multicast

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by kquinn1204** [comment]
LGTM from @mlguerrero12

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -403,6 +403,13 @@ The following updates have been made to the status management of `EgressFirewall
 
 With this update, MetalLB exposes additional metrics relating to communication between MetalLB and Border Gateway Protocol (BGP) peers. For more information, see xref:../networking/metallb/metallb-troubleshoot-support.adoc#nw-metallb-metrics_metallb-troubleshoot-support[MetalLB metrics for BGP and BFD].
 
+[id="ocp-4-15-all-multicast"]
+==== Supporting all-multicast mode
+
+{product-title}release now supports configuring the all-multicast mode by using the tuning CNI plugin. This update eliminates the need to grant the `NET_ADMIN` capability to the pod's Security Context Constraints (SCC), enhancing security by minimizing potential vulnerabilities for your pods.
```
Comment (stevsmit) [inline release_notes/ocp-4-15-release-notes.adoc:409]: Is there a reason to have release here? Or maybe you're looking to add the version attribute? {product-version}

```suggestion
{product-title} now supports configuring the all-multicast mode by using the tuning CNI plugin. This update eliminates the need to grant the `NET_ADMIN` capability to the pod's Security Context Constraints (SCC), enhancing security by minimizing potential vulnerabilities for your pods.
```

**Pull request merged**